blueflood-http/src/main/java/com/rackspacecloud/blueflood/inputs/handlers/HttpMetricsIngestionHandler.java

```diff
@@ -120,38 +133,19 @@ public void handle(ChannelHandlerContext ctx, HttpRequest request) {
                 log.warn("Exception parsing content", e);
                 DefaultHandler.sendResponse(ctx, request, "Cannot parse content", HttpResponseStatus.BAD_REQUEST);
                 return;
-            } catch (IOException e) {
-                log.warn("IO Exception parsing content", e);
-                DefaultHandler.sendResponse(ctx, request, "Cannot parse content", HttpResponseStatus.BAD_REQUEST);
-                return;
-            } catch (Exception e) {
-                log.warn("Other exception while trying to parse content", e);
-                DefaultHandler.sendResponse(ctx, request, "Failed parsing content", HttpResponseStatus.INTERNAL_SERVER_ERROR);
-                return;
-            }
-
-            List<Metric> metrics;
-            try {
-                metrics = jsonMetricsContainer.getValidMetrics();
-                forceTTLsIfConfigured(metrics);
-
-                if (jsonMetricsContainer.areDelayedMetricsPresent()) {
-                    Tracker.getInstance().trackDelayedMetricsTenant(tenantId, jsonMetricsContainer.getDelayedMetrics());
-                }
```

Comment:
Note here that I have combined all the catch statements into try with all the catches, since we're processing the json as we create the container, and not have a separate json parsing and then a toMetrics() call to process like before.